content/posts/bayestestR_presentation.Rmd

```diff
@@ -67,15 +67,15 @@ Results can also be plotted, to get a better impression of the posterior distrib
 plot(equivalence_test(model))
 ```
 
+- **Don't forget to check out the** [**documentation here**](https://github.com/easystats/bayestestR#documentation) **for more!**
 
-More details about `bayestestR`'s features are comming soon, stay tuned ;)
 
-- **Don't forget to check out the** [**documentation here**](https://github.com/easystats/bayestestR#documentation) **for more!**
+More details about `bayestestR`'s features are comming soon, stay tuned ;)
 
 
-## Help us
+## get involved
```

Comment:
Great change here! Should it be capitalized ("Get Involved")? Not sure how you're doing the other subtitles. But I like "get involved" much better, for whatever that's worth.

Reply:
fixed thanks :)